apps/web-mzima-client/src/app/shared/components/company-info/company-info.component.scss

```diff
@@ -116,9 +116,13 @@
 .mat-expansion-panel {
   &:not(.mat-expanded):not([aria-disabled='true']) {
     .mat-expansion-panel-header {
+      &:focus {
+        outline: 1px solid #fdfdfd;
+      }
```

Comment:
Because it's inside `&:not(.mat-expanded):not([aria-disabled='true'])`, it would only work after the dropdown is expanded or so.
You can move it outside of it @kivuvarosekivuvan

Reply:
Also let's use the browser default outline instead for consisitency.